code/components/discord/src/DiscordPresence.cpp

```diff
@@ -89,6 +111,8 @@ static InitFunction initFunction([]()
 	OnKillNetworkDone.Connect([]()
 	{
 		g_richPresenceOverride = "";
+		NativeInvoke::Invoke<HashString("SET_DISCORD_APP_ID"),char*,char*>(DEFAULT_APP_ID);
```

Comment:
Unsure whether it's a good idea to call natives inline like that from here.

Reply:
yeah the idea was to avoid repetitions in the code, but sure, i can fix this :)